pennylane/devices/default_qubit.py

```diff
@@ -412,6 +412,36 @@ def _create_basis_state(self, index):
     def state(self):
         return self._flatten(self._pre_rotated_state)
 
+    def density_matrix(self, wires):
+        """Returns the reduced density matrix of a given set of wires.
+
+        Args:
+            wires (Sequence[int] or int): the wires of the subsytem.
+
+        Returns:
+            tensor[complex]: complex tensor of shape ``2^(self.num_wires-len(wires))``
+            representing the reduced density matrix.
```

Comment:
Would be good to make sure it's the same as for default mixed, excepting the array size.

How about we just write the one docstring in `QubitDevice` and don't mention the output shape (which as far as I can tell is the only reason to have this repetition)